Tools/YamlCreate.ps1

```diff
@@ -753,17 +753,18 @@ Function Read-NestedInstaller {
       if ($_EffectiveType -eq 'portable') {
         do {
           Write-Host -ForegroundColor 'Red' $script:_returnValue.ErrorString()
-          Write-Host -ForegroundColor 'Green' -Object '[Required] Enter the portable command alias'
-          if (Test-String -not $_Alias -IsNull) { Write-Host -ForegroundColor 'DarkGray' "Old Variable: $_Alias" }
+          Write-Host -ForegroundColor 'Yellow' -Object '[Optional] Enter the portable command alias'
+          if (Test-String -not "$($_InstallerFile['PortableCommandAlias'])" -IsNull) { Write-Host -ForegroundColor 'DarkGray' "Old Variable: $($_InstallerFile['PortableCommandAlias'])" }
           $_Alias = Read-Host -Prompt 'PortableCommandAlias' | TrimString
           if (Test-String -not $_Alias -IsNull) { $_InstallerFile['PortableCommandAlias'] = $_Alias }
+          elseif (Test-String -not $_InstallerFile['PortableCommandAlias'] -IsNull) { $_InstallerFile['PortableCommandAlias'] = "$($_InstallerFile['PortableCommandAlias'])" }
```

Comment:
This line doesn't make sense to me. Reading through the logic . . .
If the `InstallerFile.PortableCommandAlias` is not null, set `InstallerFile.PortableCommandAlias` to `InstallerFile.PortableCommandAlias`